agreement/proposal.go

```diff
@@ -19,6 +19,7 @@ package agreement
 import (
 	"context"
 	"fmt"
+	"github.com/algorand/go-algorand/data/transactions"
```

Comment:
The agreement service should not be familiar with the concept of "transactions". Let upstream components to provide you with two versions of the block.